src/ast-lexer.cc

```diff
@@ -114,11 +116,11 @@ static WabtResult fill(WabtLocation* loc,
     /* TODO(binji): could just alloc instead, because we know we'll need to
      * memmove below */
     char* new_buffer =
-        static_cast<char*>(wabt_realloc(lexer->buffer, new_buffer_size));
+        static_cast<char*>(realloc(lexer->buffer, new_buffer_size));
```

Comment:
Not stupid, I was being lazy and just relying on the compiler to tell me when it was ambiguous. But you're right, it's better to give it a different name so it's clear. Ultimately we'll remove these functions for new/delete, but better to have it clear now.